modules/core/src/utils/pixel-ratio.js

```diff
@@ -0,0 +1,7 @@
+/* global window */
```

Comment:
Nit: Since just one default export, maybe align file name `get-pixel-ratio`?

Reply:
Can we just put this into `utils/globals.js`?

Reply:
Looks like globals.js is focusing on the gap between node and browser envs, maybe not a proper place for get-pixel-ratio